cmd/idracsdkapps/splunkapp/main.go

```diff
@@ -144,7 +146,8 @@ func logToSplunk(events []*SplunkEvent) {
 	if resp.Body != nil {
 		resp.Body.Close()
 	}
-	log.Printf("Sent to Splunk. Got back %d", resp.StatusCode)
```

Comment:
We'll want to keep the log method.
Can you change fmt.Print* to log.Print*?

Reply:
Good catch. Changed it and pushed the change.